homeassistant/scripts/check_config.py

```diff
@@ -88,6 +92,9 @@ def run(script_args: List) -> int:
         '-s', '--secrets',
         action='store_true',
         help="Show secret information")
+    parser.add_argument(
+        '--new', nargs='?', const=True, default=False,
+        help="Proof of concept config_checker")
```

Comment:
Can be simplified :)

```python
parser.add_argument('--new', action='store_true', help="...")
```

Reply:
Thanks, it will be removed eventually, but good to know :-)